cns/restserver/internalapi.go

```diff
@@ -337,7 +337,7 @@ func (service *HTTPRestService) CreateOrUpdateNetworkContainerInternal(
 	existingNCInfo, ok := service.getNetworkContainerDetails(req.NetworkContainerid)
 	if ok {
 		existingReq := existingNCInfo.CreateNetworkContainerRequest
-		if reflect.DeepEqual(existingReq.IPConfiguration, req.IPConfiguration) != true {
+		if !reflect.DeepEqual(existingReq.IPConfiguration, req.IPConfiguration) {
 			logger.Errorf("[Azure CNS] Error. PrimaryCA is not same, NCId %s, old CA %s, new CA %s", req.NetworkContainerid, existingReq.PrimaryInterfaceIdentifier, req.PrimaryInterfaceIdentifier)
```

Comment:
I saw a lot of [Azure CNS], etc. I am not sure, but I think the purpose of this label is to find where the error is from. Why not moving to `klog`. In NPM, we are gradually moving to `klog`. It gives file name and line numbers for free. So it is much easy to maintain the code.

Reply:
yes, we should standardize logging and get rid of our custom logging packages (I would actually prefer we move all our projects to [`zap`](https://github.com/uber-go/zap)). out of scope here, though.